datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -238,6 +244,8 @@ def create_scraper_configuration(self, instance=None):
         # The service account bearer token to be used for authentication
         config['_bearer_token'] = self._get_bearer_token(config['bearer_token_auth'], config['bearer_token_path'])
 
+        config['telemetry_enable'] = is_affirmative(instance.get('telemetry', default_instance.get('telemetry', False)))
```

Comment:
I suggest the naming `telemetry_enabled`. Also, I believe you're checking `telemetry` here instead of `telemetry_enable`.